src/common/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(common STATIC common.c)
```

Comment:
`s/common/${LIB_COMMON_STATIC}`, and move the `set(LIB_COMMON_STATIC common)` in the proper place (if needed).